docs/rst/use_cases/ros_cloud.rst

```diff
@@ -0,0 +1,144 @@
+
+To launch the local router, execute:
+
+.. code-block:: bash
+
+    ddsrouter -c local-ddsrouter.yaml
+
+
+Talker
+------
+This example will make use of *ROS 2 galactic* with ``demo-nodes-cpp`` package installed. If not already present in your
+system, you may choose any of the available options to `install ROS galactic <https://docs.ros.org/en/galactic/Installation.html>`_,
+or even consider directly using a distributed `Docker image <https://hub.docker.com/_/ros>`_. Just make sure the
+resulting environment is prepared to utilize |efastdds| as middleware (see `Working with eProsima Fast DDS <https://docs.ros.org/en/galactic/Installation/DDS-Implementations/Working-with-eProsima-Fast-DDS.html>`_).
+
+Once *ROS 2* is installed, start publishing messages in DDS domain ``0`` by executing:
+
+.. code-block:: bash
+
+    RMW_IMPLEMENTATION=rmw_fastrtps_cpp ros2 run demo_nodes_cpp talker
+
+
+
+Kubernetes setup
+================
+Two different deployments will be used for this example, each in a different |k8s| pod. The |ddsrouter| cloud instance
+(cloud router) consists of two participants; a WAN participant that receives the messages coming from our LAN through
+the aforementioned UDP communication channel, and a
+:ref:`Local Discovery Server <user_manual_participants_local_discovery_server>` (local DS) that propagates them to a
+ROS 2 listener node hosted in a different |k8s| pod. The choice of a Local Discovery Server instead of a Simple
+Participant to communicate with the listener has to do with the difficulty of enabling multicast routing in cloud
+environments.
+
+The described scheme is represented in the following figure:
+
+.. figure:: /rst/figures/vulcanexus_cloud.png
+
+In addition to the two mentioned deployments, two |k8s| `services <https://kubernetes.io/docs/concepts/services-networking/service/>`_
+are required in order to direct dataflow to each of the pods. A LoadBalancer will forward messages reaching the cluster
+to the WAN participant of the cloud router, and a ClusterIP service will be in charge of delivering messages from the
+local DS to the listener pod. Following are the settings needed to launch these services in |k8s|:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/load-balancer-service.yaml
+    :language: yaml
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/local-service.yaml
+    :language: yaml
+
+.. note::
+
+    An `Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`_ needs to be configured for the
+    LoadBalancer service to make it externally-reachable. In this example we consider the assigned public IP address to
+    be ``2.2.2.2``.
+
+The configuration file used for the cloud router will be provided by setting up a `ConfigMap <https://kubernetes.io/docs/concepts/configuration/configmap/>`_:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/ConfigMap.yaml
+    :language: yaml
+
+Following is represented the overall configuration of our |k8s| cluster:
```

Comment:
Doubt: is this correct in English? And the subject?

Reply:
Probably not, I changed it :+1: